evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -311,12 +307,116 @@ fn remove_file(state: &mut Cheatcodes, path: impl AsRef<Path>) -> Result<Bytes,
     Ok(Bytes::new())
 }
 
+/// Creates a new, empty directory at the provided path.
+///
+/// If `recursive` is true, it will also create all the parent directories if they don't exist.
+///
+/// # Errors
+///
+/// This function will return an error in the following situations, but is not limited to just these
+/// cases:
+///
+/// - User lacks permissions to modify `path`.
+/// - A parent of the given path doesn't exist and `recursive` is false.
+/// - `path` already exists and `recursive` is false.
+fn create_dir(state: &Cheatcodes, path: impl AsRef<Path>, recursive: bool) -> Result<Bytes, Bytes> {
+    let path =
+        state.config.ensure_path_allowed(path, FsAccessKind::Write).map_err(error::encode_error)?;
+    if recursive { fs::create_dir_all(path) } else { fs::create_dir(path) }
+        .map(|()| Bytes::new())
+        .map_err(error::encode_error)
+}
+
```

Comment:
I expect `fs` to grow eventually so would like to move all of them to new file `fs.rs`

Reply:
done in f9eb363